CHANGELOG.md

```diff
@@ -39,6 +39,18 @@ Bottom level categories:
 
 ## Unreleased
 
+### Major Changes
+
+#### Vendored WebGPU Bindings from `web_sys` 
+
+**`--cfg=web_sys_unstable_apis` is no longer needed in your `RUSTFLAGS` to compile for WebGPU!!!**
+
+While WebGPU's javascript api is stable in the browsers, the `web_sys` bindings for WebGPU are still improving. As such they are hidden behind the special cfg `--cfg=web_sys_unstable_apis` and are not available by default. Everyone who wanted to use our WebGPU backend needed to enable this cfg in their `RUSTFLAGS`. This was very inconvenient and made it hard to use WebGPU, especially when WebGPU is enabled by default. Additionally, the unstable APIs don't adhere to semver, so there were repeated breakages.
```

Comment:
The JS API is stable? Or what does "stable in the browsers" mean here exactly?

If the JS API is really stable we could potentially stabilize it in `web-sys` as well, but my assumption was that it is indeed not stable.

Reply:
To a first order approximation webgpu is stable, yes. It's a bit more complicated in practice, and I don't think web-sys can stabilize for a while yet.
- All programs that work today on a stable browser will continue to work.
- Features that have not shipped on a stable browser, may yet change.
  - `write_timestamp` was removed fairly recently, this was before chrome shipped the timestamp feature.
- the IDL has no stability guarentees - webgpu hasn't been standardized yet. Things like names of interfaces may change, so long as the js "abi" doesn't.